### PR TITLE
Exercise 1 + extra credit

### PR DESCRIPTION
**Pull request by benjmyers**

### CODE REVIEW

File: src/exercise/01.js [hunk 1]
```diff
@@ -3,12 +3,13 @@
...
   // 💣 delete this variable declaration and replace it with a React.useState call
-  const name = ''
+  const [name, setName] = React.useState(initialName);
 
   function handleChange(event) {
     // 🐨 update the name here based on event.target.value
+    setName(event.target.value);
   }
 
   return (
```
Comment (nikkifurls) [inline src/exercise/01.js:15]: I _think_ you'll also want to add `value={name}` to the `input` here :)

**Review by nikkifurls — APPROVED** [review]
🦘

File: src/exercise/01.js (hunk 1 above)
Comment (benjmyers) [reply src/exercise/01.js:15]: Hm that's kinda weird because it works without it!?

Comment (nikkifurls) [reply src/exercise/01.js:15]: Really?!? Well that's good to know! Thank you!

Nothing makes sense in this world LOL 🤦

**Pull request merged**